elide-spring/elide-spring-boot-autoconfigure/src/test/java/com/yahoo/elide/contrib/dynamicconfig/model/PlayerStats.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.contrib.dynamicconfig.model;
+
+import com.yahoo.elide.annotation.DeletePermission;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.annotation.UpdatePermission;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+import com.yahoo.elide.datastores.aggregation.annotation.CardinalitySize;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+
+import java.util.Date;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+/**
+ * A root level entity for testing AggregationDataStore.
+ */
+@Include(rootLevel = true , type = "playerStats")
+@Cardinality(size = CardinalitySize.LARGE)
+@EqualsAndHashCode
+@ToString
+@Data
+@Entity
+@ReadPermission(expression = "Prefab.Role.All")
+@UpdatePermission(expression = "Prefab.Role.None")
+@DeletePermission(expression = "Prefab.Role.None")
+public class PlayerStats {
```

Comment:
elide-example-models was a place to put common classes that are referenced in multiple maven sub-modules.  Rather than redefine these entities here - can we move Player stats and related models to elide-example-models?  That said, I don't want this to cascade into a ton of work - so let's discuss if this looks complicated.

Reply:
We will take care of this once all these PRs are merged into a single PR.